docs/examples/use_cases/tensorflow/efficientdet/README.rst

```diff
@@ -0,0 +1,124 @@
+EfficientDet with TensorFlow and DALI
+=================================
```

Comment:
Please make this `README` better. Let's be more user-friendly and when user navigates here, he should get some useful information. I don't expect this README to be like the ones in [Deep Learning Examples](https://github.com/NVIDIA/DeepLearningExamples/blob/master/PyTorch/SpeechRecognition/Jasper/README.md), but I do believe README is one of the most important parts of the example.

Reply:
This was based on other README pages we have for our other use cases. How about we figure out some concrete ideas on how to make docs better for this whole section and do it as a separate effort?

Reply:
Fine by me. Just pointing out review issues I see ;)